src/models/m_rsc.erl

```diff
@@ -191,7 +192,7 @@ page_path_to_id(Path, Context) ->
 
 
 %% @doc Read a whole resource, check all properties for access rights
--spec get_visible(resource(), #context{}) -> list() | undefined.
+-spec get_visible(resource(), #context{}) -> props() | undefined.
```

Comment:
The spec in line 195 uses a record, please define a type for the record and use that instead.